evap/staff/importers.py

```diff
@@ -350,14 +356,14 @@ def check_data_type_correctness(self):
         """
         Checks that all cells after the skipped rows contain string values (not floats or integers).
         """
-        for sheet in self.book.sheets():
-            for row in range(self.skip_first_n_rows, sheet.nrows):
-                if not all(isinstance(cell, str) for cell in sheet.row_values(row)):
+        for sheet in self.book:
+            for row_idx, row in enumerate(sheet.iter_rows(values_only=True)):
```

Comment:
The old code started at `skip_first_n_rows` (matching the comment). This would also check the header rows, which I think we do not want.